src/server/scripts/EasternKingdoms/zone_hinterlands.cpp

```diff
@@ -49,18 +49,18 @@ enum Rinji
     GO_RINJI_CAGE           = 142036
 };
 
-struct Location
+struct LocationXYZ
 {
-    float posX, posY, posZ;
+    float x, y, z;
```

Comment:
This for example

Reply:
only this is not related, the other changes are for the cppcheck reports

I saw that LocationXYZO is more compact, and writing `pos` is unnecessary considering that we already talking about a "location", so I removed the `pos` to be more coherent with the `LocationXYZO`.

Hope it is not an issue, I will try to reduce the changes as much as possible